aspnetcore/host-and-deploy/iis/index.md

```diff
@@ -527,14 +527,14 @@ To set environment variables for individual apps running in isolated app pools (
 
 ## Configuration sections of web.config
 
-Configuration sections of ASP.NET 4.x apps in *web.config* aren't used by ASP.NET Core apps for configuration:
+Configuration sections of ASP.NET apps in *web.config* aren't used by ASP.NET Core apps for configuration:
 
 * `<system.web>`
 * `<appSettings>`
 * `<connectionStrings>`
 * `<location>`
 
-ASP.NET Core apps are configured using other configuration providers. For more information, see [Configuration](xref:fundamentals/configuration/index).
+ASP.NET Core apps are configured using other configuration providers. For more information, see [Configuration](xref:fundamentals/configuration/index) and [.NET Core run-time configuration settings](dotnet/core/run-time-config/)
```

Comment:
```suggestion
ASP.NET Core apps are configured using other configuration providers. For more information, see [Configuration](xref:fundamentals/configuration/index) and [.NET Core run-time configuration settings](xref:dotnet/core/run-time-config/)
```

Reply:
```suggestion
ASP.NET Core apps are configured using other configuration providers. For more information, see [Configuration](xref:fundamentals/configuration/index) and [.NET Core run-time configuration settings](/dotnet/core/run-time-config/)
```

Reply:
Hmm, just occurred to me that I'm not sure how xref works across repos, so I may have that wrong, looking for an example...

Reply:
```suggestion
ASP.NET Core apps are configured using other configuration providers. For more information, see [Configuration](xref:fundamentals/configuration/index) and [.NET Core run-time configuration settings](/dotnet/core/run-time-config/)
```

Reply:
I just forgot the leading `/`